docs/test-results/viewing-test-results.md

```diff
@@ -83,10 +84,11 @@ Your test assets, including videos, screenshots, and logs, are retained for 30 d
 1. In Sauce Labs, in the left panel, click **AUTOMATED** or **LIVE** and then click **Test Results**.
 2. Click the test whose results you want to view.
 3. On the **Test Details** page, you can do the following:
-  * Click the play button to view screenshots and video of the test. Audio is not recorded.
   * Click the **COMMANDS** tab to see the API commands that were issued during the test. The **COMMANDS** tab also includes controls for filtering the commands displayed and playing back the commands.
```

Comment:
The sentences for each bulleted point could also be updated so the focus is on what the value is for the user. For example: "To see the API commands that were issued during the test, click the **Commands** tab. (...)